.depend

```diff
@@ -366,7 +363,7 @@ parsing/parser.cmi : \
     parsing/parsetree.cmi \
     parsing/location.cmi \
     parsing/docstrings.cmi \
-    parsing/camlinternalMenhirLib.cmi
+    parsing/camlinternalMenhirLib.cmo
```

Comment:
@jhjourdan these camlinternalMenhirLib-related .depend changes look wrong to me, could you remove them from the diff? (Adding `path.cmi` to includecore is probably fine, but the rest should probably go away.)

Reply:
I reverted all my changes to `.depend`: This PR does not touch the compiler, so its `.depend` file should not change.

That said, if what you say is right, then there probably is an issue with `make depend`, since all I did for updating `.depend` is running `make depend`...